DOCS-5460 Update organizations overview to better explain orgs and org features (#873)
clerk/clerk-docs: docs/organizations/overview.mdx

There is no limit to the number of organizations a user can be a member of.

However, there is a limit to how many members total can be in a single organization. By default, the membership limit is set to 3 members. To change this limit, scroll to the **Default membership limit** section and update the membership limit.

If you are on the Free plan, you can update the membership limit to a maximum of 5 members.

If you have the Pro plan, you can set the membership limit to unlimited.

You can also change this limit on a per-organization basis:

1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations).
2. In the navigation sidebar, select **Organizations**.
3. Select the organization you want to update.
4. In the **Membership limit** section, update the membership limit.

### Default ability to delete

By default, organizations are deletable. Any member with the "Delete organization" permission can delete an organization. To prevent organizations from being deleted, you can disable the ability to delete organizations by scrolling to the **Default ability to delete** section and unchecking the option.

### Verified domains

> Verified domains can be used to streamline enrollment into an organization. For example, if the domain `@clerk.com` is verified, any user with an email address ending in `@clerk.com` can be automatically invited or be suggested to join an organization with that domain. This feature is useful for organizations that want to restrict membership to users with specific email domains. See the [verified domains](/docs/organizations/verified-domains) documentation for more information.

Reviewer: ```suggestion
Verified domains can be used to streamline enrollment into an organization. For example, if the domain `@clerk.com` is added to an organization, any user with a `@clerk.com` email address can be automatically invited or be suggested to join this organization. This feature is useful for organizations that want to restrict membership to users with specific email domains. See the [verified domains](/docs/organizations/verified-domains) documentation for more information.
````

## Active organization

> There are two types of accounts that a user can have in a Clerk application: a personal account and an organization account. A personal account is the default account type, and it is created when a user signs up for your application. An organization account is created when a user creates or joins an organization within your application.

Reviewer: Technically there are not two types of accounts. It's a single account that operates under different contexts, personal and organization related. We should make sure that customers don't get confused and think those are separated accounts

### User membership management

Clerk provides a set of prebuilt components to help your users manage their organization memberships:

- [`<CreateOrganization />`](/docs/components/organization/create-organization) - A form for a user to create a new organization.
- [`<OrganizationProfile />`](/docs/components/organization/organization-profile) - A profile page for the user's currently active organization. Out of the box, this component's **General** tab displays the organization's information and the **Leave organization** option. The **Members** tab shows the organization's members along with their join dates and roles.
- [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher) - A dropdown menu that allows a user to switch between their personal account and their organization account(s).
- [`<OrganizationList />`](/docs/components/organization/organization-list) - A list of organizations that a user is a member of.

> ### Admin membership management

Reviewer: What are these about? Are they Dashboard operations or you mean the available Backend API operations?

As an admin, you can manage organization memberships

- Membership list
- Domains
- Invitations (invite form, invitations list)
- Membership suggestions
- Membership requests

## Frequently asked questions (FAQ)

### How many organizations can a user belong to?

There is no limit to the number of organizations a user can be a member of.

### How many organizations can my users create?

A user within one of your applications can create up to 100 organizations in that application. This limit is separate from the number of organizations *you* can create across your entire account.

> ### How many organizations can I create?

Reviewer: In a dev instance you can create up to 50 organizations. There is no limit for production instances.
After that you will be charged in a different way depending on how many free MAOs your current plan comes with
